cap/modules/mail/ext.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not needed? Should be initialized from https://github.com/inveniosoftware/invenio-mail/blob/master/invenio_mail/ext.py